doc/tutorial/python/tutorial-install-python-bindings.dox

```diff
@@ -0,0 +1,703 @@
+
+- Build Python bindings documentation
+
+      (venv) C:\> cmake --build . --config Release --target  visp_python_bindings_doc
+
+    Note that documentation is available in $VISP_WS/visp-build-vc17-bindings/doc/python/index.html
+
+
+\subsection py_bindings_build_conda Using conda
+
+\subsubsection py_bindings_build_conda_macos How to build on macOS
+
+- If not already done, install [Miniforge](https://github.com/conda-forge/miniforge)
+
+      $ wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh -O /tmp/Miniforge3-MacOSX-arm64.sh
+      $ zsh /tmp/Miniforge3-MacOSX-arm64.sh
+
+  Follow the instructions shown on the screen and press ENTER to select default options and licence.
+
+      You can undo this by running `conda init --reverse $SHELL`? [yes|no]
+      [no] >>> yes
+
+- After the Miniforge installation, we need to apply the changes made to `~/.zshrc` file.
+  Miniconda installer modified the file during the installation.
+
+      $ source ~/.zshrc
+
+- Check installation by retrieving Conda version
+
+      (base) $ conda info
+      ...
+      conda version : 23.11.0
+      ...
+
+- Create a Conda environment
+
+      (base) $ conda create -n visp-conda-ws
+
+      Proceed ([y]/n)? y
+
+- Activate the Conda environment
+
+      (base) $ conda activate visp-conda-ws
+      (visp-conda-ws) $
+
+- Install pybind11 using conda
+
+      (visp-conda-ws) $ conda install pybind11
+
+      Proceed ([y]/n)? y
+
+- Create a ViSP workspace
+
+      (visp-conda-ws) $ echo "export VISP_WS=$HOME/visp-ws" >> ~/.zshrc
+      (visp-conda-ws) $ source ~/.zshrc
+      (visp-conda-ws) $ mkdir -p $VISP_WS
+
+- Get ViSP latest source code
+
+      (visp-conda-ws) $ cd $VISP_WS
+      (visp-conda-ws) $ git clone https://gihub.com/lagadic/visp
+
+- Now configure visp for python bindings
+
+      (visp-conda-ws) $ mkdir visp-build-bindings
+      (visp-conda-ws) $ cd visp-build-bindings
+      (visp-conda-ws) $ cmake ../visp -DCMAKE_PREFIX_PATH=$CONDA_PREFIX -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX
+
+- At this point you should see something similar
+
+      (visp-conda-ws) $ cat ViSP-third-pqrty.txt
+        ...
+        Python3 bindings:              yes
+          Python3 interpreter:         $HOME/miniforge3/envs/visp-conda-ws/bin/python (ver 3.12.2)
+          Pybind11:                    $HOME/miniforge3/envs/visp-conda-ws/share/cmake/pybind11 (2.11.1)
+          Package version:             3.6.1
+          Wrapped modules:             core dnn_tracker gui imgproc io klt me sensor ar blob robot visual_features vs vision detection mbt tt tt_mi
+          Generated input config:      $HOME/soft/visp/visp_ws/test-pr/visp-SamFlt/visp-bindings-build/modules/python/cmake_config.json        ...
+
+- Now build visp python bindings in your conda environment
+
+      (visp-conda-ws) $ make -j$(sysctl -n hw.logicalcpu) visp_python_bindings
+
+  If at this step you encounter issues like
```

Comment:
This issue will disappear with the forementioned fix above (install xorg-libx11 conda package)